### PR TITLE
Convert the output type of bool operation to int

### PR DESCRIPTION
**Pull request by inocsin**
# Description
There is a case 
```
code
  x1 = torch.to(torch.ge(x0, 0.5), torch.device("cuda:0"), 3, False, False)
  _136 = torch.sum(torch.mul(x1, CONSTANTS.c0), [1], False)

graph
  %1284 : Tensor = prim::Constant[value=<Tensor>]()
  %1275 : Tensor = aten::ge(%x0.1, %1274)
  %x1.1 : Tensor = aten::to(%1275, %1315, %1278, %484, %484, %486)
  %1285 : Tensor = aten::mul(%x1.1, %1284)
```
The output of aten::ge is Bool type, and there are three way to support the above graph
1. implement aten::to, to convert datatype from Bool to Int
2. convert datatype in aten::mul (see https://github.com/NVIDIA/TRTorch/pull/321), but I found the trtorch framework doesn't support specifying input data type, so I cann't test this method
3. convert the output type of Boolean Operation (that's what this PR does)

## Type of change

- New feature (non-breaking change which adds functionality)
Convert ouput type from bool to int

# Checklist:

- [x] My code follows the style guidelines of this project (You can use the linters)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests to verify my fix or my feature
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by narendasan** [comment]
Can you write test cases for this?

**Comment by inocsin** [comment]
> Can you write test cases for this?

I think we already have test cases for boolean operation

**Comment by developer0hye** [comment]
Does this code only works when we set precision to INT?

When I update my code with your code, 
image

**Comment by developer0hye** [comment]
Please check my pull #350

**Comment by peri044** [comment]
We now support aten::to operator which handles such masked multiplications. Closing this PR
